gpiozero/rotary_encoder.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+from gpiozero import Device, DigitalInputDevice, Button, CompositeDevice
+
+
+class RotaryEncoder(Device):
```

Comment:
Extending Device is quite low-level. I guess it doesn't get any source/values stuff?